src/webui/index.js

```diff
@@ -145,26 +134,12 @@ module.exports = async function (ctx) {
     })
   })
 
-  // Avoid setting CORS by acting like /webui loaded from API port
+  // Set user agent
   session.defaultSession.webRequest.onBeforeSendHeaders((details, callback) => {
-    details.requestHeaders.Origin = apiOrigin(apiAddress)
     details.requestHeaders['User-Agent'] = `ipfs-desktop/${VERSION} (Electron ${ELECTRON_VERSION})`
```

Comment:
:information_source:  Presence of `Electron` here makes  https://github.com/ipfs/go-ipfs-cmds/pull/201/files#diff-9cd5b98b32603671d06faab6568b1be0d3f2790b8ee426270cb20cb91824997aR182 allow the requests.